utils/servers/simplehttpserver.js

```diff
@@ -30,14 +30,34 @@ var port = 8000,
 		"mp4": "video/mp4",
 		"txt": "text/plain",
 		"bin": "application/octet-stream"
-	};
+  };
+
+// https://github.com/parshap/node-sanitize-filename/blob/master/index.js#L33-L47
+var illegalRe = /[\?<>:\*\|":]/g;
+var controlRe = /[\x00-\x1f\x80-\x9f]/g;
+var reservedRe = /^\.+$/;
+var windowsReservedRe = /^(con|prn|aux|nul|com[0-9]|lpt[0-9])(\..*)?$/i;
+var windowsTrailingRe = /[\. ]+$/;
+
+function sanitize(input) {
```

Comment:
It would be good if `sanitize()` would also respect the code style.

Reply:
this now also follows the code style